packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx

```diff
@@ -9,22 +9,44 @@
 import React, { useMemo } from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiCallOut } from '@elastic/eui';
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
 import { MaintenanceWindowStatus, KibanaServices } from './types';
 import { useFetchActiveMaintenanceWindows } from './use_fetch_active_maintenance_windows';
 
 const MAINTENANCE_WINDOW_FEATURE_ID = 'maintenanceWindow';
-const MAINTENANCE_WINDOW_RUNNING = i18n.translate(
-  'alertsUIShared.maintenanceWindowCallout.maintenanceWindowActive',
-  {
-    defaultMessage: 'Maintenance window is running',
-  }
-);
 const MAINTENANCE_WINDOW_RUNNING_DESCRIPTION = i18n.translate(
   'alertsUIShared.maintenanceWindowCallout.maintenanceWindowActiveDescription',
   {
     defaultMessage: 'Rule notifications are stopped while the maintenance window is running.',
```

Comment:
>    defaultMessage: 'Rule notifications are stopped while the maintenance window is running.',

Per my other suggestion, to make this appropriate for situations where multiple windows are running, here are some alternatives:

- Rule notifications are stopped while maintenance windows are running.
- Rule notifications are stopped during maintenance windows.